frontend/src/components/search/SearchCallToAction.tsx

```diff
@@ -7,19 +7,20 @@ import { SEARCH_CRUMBS } from "../../constants/breadcrumbs";
 
 const SearchCallToAction: React.FC = () => {
   return (
-    <GridContainer>
-      {/* TODO: BetaAlert Breadcrumbs */}
+    <>
```

Comment:
The `Breadcrumbs` component shouldn't be inside the `GridContainer`, as the component has one itself. Nesting a `GridContainer` causes unnecessary extra padding